### PR TITLE
Feat/met 754 duplicate statement checking during normalizaion

### PR DESCRIPTION
**Pull request by jochen-vermeulen**

### CODE REVIEW

File: metis-normalization/src/main/java/eu/europeana/normalization/normalizers/RemoveDuplicateStatementNormalizer.java [hunk 1]
```diff
@@ -48,11 +55,9 @@ private static final XpathQuery getProxySubtagQuery(Namespace.Element subtag) {
   @Override
   public NormalizationReport normalize(Document edm) throws NormalizationException {
     final InternalNormalizationReport report = new InternalNormalizationReport();
-    final List<Element> elements = new ArrayList<>();
-    final Set<TextLanguagePair> foundPairs = new HashSet<>();
     for (XpathQuery[] fieldSet : FIELD_SETS_TO_EVALUATE) {
 
-      elements.clear();
```
Comment (stzanakis) [inline metis-normalization/src/main/java/eu/europeana/normalization/normalizers/RemoveDuplicateStatementNormalizer.java]: Why did you change that, does it cause a problem? I did this to fix the technical debt issues..(these issues are not visible on the IDE) :D
Some below

Comment (jochen-vermeulen) [reply metis-normalization/src/main/java/eu/europeana/normalization/normalizers/RemoveDuplicateStatementNormalizer.java]: I fixed it.
